### PR TITLE
upgrades to bring all relay things in the same versions

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **Enhancement**

This PR resolves [CX-]

### Description

I was reading the docs and saw that all these packages except the relay language typescript, are released together, so they should be in the same versions.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -127,7 +127,6 @@
     "react-use": "^17.1.0",
     "redux-logger": "3.0.6",
     "relay-hooks": "3.5.2",
-    "relay-runtime": "^10.0.1",
```
Comment (damassi) [inline package.json:130]: This not needed?

**Review by damassi — APPROVED** [review]
One q about removing `relay-runtime`, but otherwise LGTM 👍

File: package.json (hunk 1 above)
Comment (pvinis) [reply package.json:130]: it is a dep of `react-relay`, so we might as well remove it. we don't use it directly.

**Comment by damassi** [comment]
#mergeOnGreen

**Pull request merged**